packages/experimental/Button/src/ButtonColorTokens.win32.ts

```diff
@@ -3,25 +3,14 @@ import { TokenSettings } from '@fluentui-react-native/use-styling';
 import { ButtonTokens } from './Button.types';
 import { shadowStyleFromGlobalToken } from './shadowStyle';
 
-export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = (t: Theme) =>
+export const defaultButtonColorTokens: TokenSettings<ButtonTokens, Theme> = (t: Theme) =>
   ({
     backgroundColor: t.colors.neutralBackground1,
     color: t.colors.neutralForeground1,
     borderColor: t.colors.neutralStroke1,
     iconColor: t.colors.neutralForeground1,
```

Comment:
Yeah, the plan is eventually all platforms would use these tokens. If I could, I would make it so that windows/win32 shared the same file, but I can only do one platform at a time with the forking. If I make two objects to share instead I believe it would load/create both objects depending on how I write the code so I figured this would be better even though there would be some code duplication. Let me know if you have other thoughts/ideas.